src/modules/ekf2/EKF/ekf.cpp

```diff
@@ -159,14 +138,52 @@ bool Ekf::update()
 		float filter_update_s = 1e-6f * _params.filter_update_interval_us;
 		_dt_ekf_avg = 0.99f * _dt_ekf_avg + 0.01f * math::constrain(input, 0.5f * filter_update_s, 2.f * filter_update_s);
 
+		// Filter accel for tilt initialization
+		if (_is_first_imu_sample) {
+			_accel_lpf.reset(imu_sample_delayed.delta_vel / imu_sample_delayed.delta_vel_dt);
+			_gyro_lpf.reset(imu_sample_delayed.delta_ang / imu_sample_delayed.delta_ang_dt);
+			_is_first_imu_sample = false;
+
+			_state = {};
+			initialiseTilt(_accel_lpf.getState());
```

Comment:
Is this to get a tilt estimate even if accel norm is > 20% longer or smaller than 1g?